Apps/WebClient/src/ClientApp/src/utility/errorTranslator.ts

```diff
@@ -22,7 +23,7 @@ export default class ErrorTranslator {
             return {
                 title,
                 description: this.getDisplayMessage(error.errorCode),
-                detail: error.resultMessage,
+                detail: `${error.resultMessage}. If this issue persists, contact HealthGateway@gov.bc.ca and provide ${error.traceId}.`,
```

Comment:
Looks like there needs to be a break-line between this one and the next message. How about adding the trace in a different field. That would need to add the field to the Banner error, but it gives us more flexibility to display it later.

Reply:
thanks @FuriousLlama , fixed!